content/espanol/security-phishing/answer-key.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Answer Key - Security"
+description: "Easy reference answer key for the security curriculum"
```

Comment:
Although this is hidden, we should translate these too

Reply:
fixed